src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,138 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+
+    static class Node<T> {
```

Comment:
```suggestion
    private static class Node<T> {
```
Also static class should be after all methods in class